schema.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE IF NOT EXISTS authors (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  first_name VARCHAR(255),
+  last_name VARCHAR(255),
+  FOREIGN KEY(item_id) REFERENCES item(id)
+);
+CREATE TABLE IF NOT EXISTS games (
+  id INT,
+  multiplayer BOOLEAN,
+  last_played_at DATE,
+  PRIMARY KEY (id),
+  FOREIGN KEY(game_id) REFERENCES games(id),
+  FOREIGN KEY(author_id) REFERENCES authors(id)
+)
```

Comment:
- [ ] Please follow the given UML diagram. Please note that the `game` class inherits from the `item` class.
This can be presented in your schema by adding a foreign key of the `item` to the `game` class.

- [ ] Please also remove any columns in your class that don't exist on the UML diagram.